report/sarif_test.go

```diff
@@ -49,6 +49,7 @@ func TestWriteSarif(t *testing.T) {
 		t.Run(test.cfgName, func(t *testing.T) {
 			tmpfile, err := os.Create(filepath.Join(t.TempDir(), test.testReportName+".json"))
 			require.NoError(t, err)
+			defer tmpfile.Close()
```

Comment:
whoops, thanks for adding this in

Reply:
Surprisingly, this didn't seem to cause trouble on *nix. It was only a consistent issue on Windows.